playground/tailwind/postcss.config.ts

```diff
@@ -1,7 +0,0 @@
-// postcss.config.ts
-module.exports = {
-  plugins: {
-    tailwindcss: { config: __dirname + '/tailwind.config.js' },
-    autoprefixer: {},
-  },
-}
```

Comment:
We wanted to test that `postcss.config.ts` work specifically, so I think we should revert the extension to `.ts`.

Reply:
Did not find a very mainstream solution. I reverted it back to CJS to keep imitating the user's config (without a hacking workaround). We could move it to CJS after the upstream updated. I left a comment in the eslintrc.